crates/picker/src/picker.rs

```diff
@@ -49,6 +49,9 @@ pub trait PickerDelegate: Sized + 'static {
     fn set_selected_index(&mut self, ix: usize, cx: &mut ViewContext<Picker<Self>>);
 
     fn placeholder_text(&self, _cx: &mut WindowContext) -> Arc<str>;
+    fn no_matches_text(&self, _cx: &mut WindowContext) -> Arc<str> {
```

Comment:
nit: this could return a SharedString instead of `Arc<str>`. Feel free to ignore it though.

Reply:
Fixed